build-packages/README.md

```diff
@@ -0,0 +1,17 @@
+# Build Packages
+
+This directory contains code for GitHub Actions that are used in this repo's workflow.
+
+The actions are built with yarn like the rest of this repo.
+
+To build / update the actions, run this inside the action's directory:
+
+```
+yarn run all
+```
+
+This will compile, lint and package the actions.
+
+Changes in the `dist` folder need to be committed to the git repository as this is the code which is being run by GitHub.
```

Comment:
[q] Is there still a `dist` folder? Seems like `dist` was replaced by `lib` and doesn't actually need to be committed anymore (see `.gitignore`).

Reply:
Forgot to update. The compiled action (which was in dist before is now in the 'root' of the action in `.github/actions/action-name`. Will update the readme, thanks for the hint.